### DISCUSSION TITLE
Use existing docker server to throw actions

### DISCUSSION THREAD

**Discussion by davidaparicioalvarez** [microsoft/AL-Go#873] [Q&A]
Hi all,

I am rookie using Al-Go for GitHub. Now I am using DevOps as external repository and local docker server to test my apps. I am migrating my repositories to GitHub and implementing they with Al-Go.

In my developments I use a docker server configured with the necessary containers, created with my own powershell script using BcContainerHelper, for each of my customers with external database server. Is it possible to use these docker server to run (not recreate) the appropriate container for the repository when throw an action?

Thaks you.

David

**Comment by freddydk** [comment]
Going through all open discussions and finding this one without an answer for 14 months...

There isn't a good way to use your own containers for the build process.
We are working on allowing online sandboxes, and maybe local containers could be enabled as well (if they are accessible over the internet)

Is local containers something that would be relevant to people?
